docs/configuration.rst

```diff
@@ -2342,20 +2342,37 @@ Can be overwritten for each single need by setting :ref:`need_collapse`.
 .. _needs_per_id_build_path:
 
 needs_per_id_build_path 
-~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Maybe this option should be also related to the builder_name option?
`needs_build_json_per_id` --> `needs_build_json_per_id_path`?